ynput/ayon-core
# Blender: Implement USD Extractor and Loader

## Changelog Description

Implement simple USD extract and load for Blender

## Additional info

This does not contain logic yet to 'contribute' the USD data into another USD Asset layer. That will be a separate PR.

## Testing notes:

1. Extract USD geometry from Blender
2. Load USD to Blender
3. Load Alembic to Blender should still work
4. Updating loaded content should work
